src/cli.ts

```diff
@@ -86,6 +86,20 @@ const {
 
 const debug = require("debug")("binance-oco");
 
+const exitHooks = (cancel: Function) => {
```

Comment:
Is it possible to implement this logic using an existing node package such as [exit-hook](https://www.npmjs.com/package/exit-hook) or [node-cleanup](https://www.npmjs.com/package/node-cleanup)?

Reply:
Did not know about those - will investigate!